src/main/webapp/dev/js/common/statusMessage.es6

```diff
@@ -24,6 +24,7 @@ function populateStatusMessageDiv(message, status) {
     $statusMessageDivContent.addClass('alert');
     // Default the status type to info if any invalid status is passed in
     $statusMessageDivContent.addClass(`alert-${StatusType.isValidType(status) ? status : StatusType.INFO}`);
+    $statusMessageDivContent.addClass(`icon-${StatusType.isValidType(status) ? status : StatusType.INFO}`);
```

Comment:
@whipermr5 Sorry, but could you also suggest a variable name. Drawing a blank on this one 😅